backend/src/main/resources/liquibase/create-versioning-tables-changelog2.xml

```diff
@@ -296,4 +296,17 @@
 				constraintName="ck_versioning_modeldb_entity_mapping"
 				tableName="versioning_modeldb_entity_mapping"/>
 	</changeSet>
+	<changeSet id="changeKeysOfFolderElement" author="lezhevg">
+		<dropTable tableName="folder_element"/>
```

Comment:
I’m not sure I follow why we need to drop the table

Reply:
fixed

Reply:
break them into changeset as mentioned in the message. A single changeset if interrupted won't rollback create table.